### PR TITLE
[EPMDEPS-13981] Remove Areas markup type

### PR DESCRIPTION
**Pull request by timnbv**
Remove all Area mentions from the codebase
Fix tests and test snapshots

### CODE REVIEW

File: src/containers/LabelingCanvas/useLabels.test.js [hunk 1]
```diff
@@ -38,9 +37,8 @@ jest.mock('@/actions/markup', () => mockMarkupActions)
 jest.mock('@/actions/ui', () => mockUiActions)
 
 const mockLabel = new Label(1, 2, 3, 4)
-const mockArea = new Area(1, 2, 3, 4)
 const mockLabels = [mockLabel]
-const mockObjects = [mockLabel, mockArea]
+const mockObjects = [mockLabel]
```
Comment (Rob-n-Dude) [inline src/containers/LabelingCanvas/useLabels.test.js:41]: looks like we can operate without `mockObjects` now, right?

File: src/containers/LabelingCanvas/useTables.test.js [hunk 2]
```diff
@@ -38,9 +37,8 @@ jest.mock('@/actions/markup', () => mockMarkupActions)
 jest.mock('@/actions/ui', () => mockUiActions)
 
 const mockTable = new Table([0.45, 0.765], [8.234, 9.564])
-const mockArea = new Area(1, 2, 3, 4)
 const mockTables = [mockTable]
-const mockObjects = [mockTable, mockArea]
+const mockObjects = [mockTable]
```
Comment (Rob-n-Dude) [inline src/containers/LabelingCanvas/useTables.test.js:41]: same as above 😃

**Comment by Rob-n-Dude** [comment]
LGTM

File: src/actions/markup.js [hunk 3]
```diff
@@ -9,7 +9,6 @@ import { documentNameSelector } from '@/selectors/document'
 import {
   markupSelector,
   pageSelectedLabelsSelector,
-  pageSelectedAreasSelector,
```
Comment (DrDiman) [inline src/actions/markup.js:12]: 1. drop AREA_BORER from theme.default as never used
2. and rename AREA_BACKGROUND to MERGE_PROJECTION_BACKGROUND

**Review by DrDiman — APPROVED** [review]
well done. Thanks 👍

File: src/actions/markup.js (hunk 3 above)
Comment (timnbv) [reply src/actions/markup.js:12]: Done

File: src/containers/LabelingCanvas/useLabels.test.js (hunk 1 above)
Comment (timnbv) [reply src/containers/LabelingCanvas/useLabels.test.js]: Fixed

File: src/containers/LabelingCanvas/useTables.test.js (hunk 2 above)
Comment (timnbv) [reply src/containers/LabelingCanvas/useTables.test.js]: Fixed

**Pull request merged**